### DISCUSSION TITLE
Simulations with CKD Stages templates AUC_inf relative to Pop healthy simulation AUC_inf Ratio

### DISCUSSION THREAD

**Discussion by sajabzughoul** [Open-Systems-Pharmacology/Forum#1046] [Questions & Problems] [answered]
I have no explanation for the decrease in ratio for stage 4 and 5 relative to stage 3 in comparison to control healthy, my drug have non-renal clearance with <1% of drug excreted unchanged in urine, Can anyone give me an explanation for that or even give me the head of the thread to search for.
Thank you

**Comment by prvmalik** [comment, marked as answer]
A number of parameters are changed vs. healthy 

Fraction unbound 
Intestinal transit 
Gastric emptying 
Hematocrit
Kidney volume 
Kidney blood flow 

Likely fraction unbound is responsible for the greatest change in a scenario with no renal CL. A sensitivity analysis would be enlightening.

Increasing fraction unbound lowers plasma concentrations.

But a model should account for changes in nonrenal clearance in CKD stages. Generally speaking for a compound as you describe AUCR should be less than 2 however.

**Reply by sajabzughoul** [reply]
Thank you very much, I appreciate your help 
Sajabzughoul 
 Master student at Department of Pharmaceutical Technology, Faculty of Pharmacy, Jordan University of
Science and Technology, Irbid, Jordan

**Comment by sajabzughoul** [comment]
The fraction unbound used in all simulation was the fraction unbound in healthy human individual. A range for fraction unbound for diseased Pop would be enough, true?
How can I do sensitivity analysis for population simulation?



Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by prvmalik** [comment]
Fraction unbound is scaled up by the plasma protein scale factor in the CKD pop.

Increased distribution into cellular spaces will enable faster clearance in addition to the volume of distribution being larger, which is why your AUC might change (though fundamental PKers will disagree on this, quite a complex topic).

For a sensitivity analysis, change your parameter of interest by +/- 10% and record the change in the output of interest. 

Paul
